lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -196,21 +197,17 @@ $progress-bar-layers: (
 	}
 
 	&__labelText {
-		@include headlineXXS();
-		@include textBold;
+    font-size: var(--typography-font-size-xs);
```

Comment:
was it intended to not use token here? Is it real use-case or something?

Reply:
It's a real use-case. This line set smaller font size for mobile.

Reply:
weh nwe introduce Typography tokens, do we plan to implement it this way? I thought, we'll be using only tokens, not variables...

Reply:
Right, in FIgma there is another token for mobile. Fixed.